app/gui/suggestion-database/src/mock.rs

```diff
@@ -380,13 +377,13 @@ macro_rules! mock_suggestion_database {
 #[macro_export]
 macro_rules! doc_section_mark {
     (!) => {
-        $crate::engine_protocol::language_server::Mark::Important
+        enso_doc_parser::Mark::Important
```

Comment:
It assumes that in every crate using this macro, the `enso_doc_parser` is also added as a dependency, isn't it? Instead, I would make it same as before: reexport enso_doc_parser in this crate and use `$crate` notation.

Same in 7 places below.